app/full_text_search.py

```diff
@@ -20,19 +20,50 @@
 
 import model
 
-# The index name for full text search
-PERSON_FULL_TEXT_INDEX_NAME = 'person_information'
+# The index name for full text search.
+# This index contains person name and location.
+PERSON_LOCATION_FULL_TEXT_INDEX_NAME = 'person_location_information'
+
+def make_or_regexp(query_txt):
+    """
+    Craetes compiled regular expression for or search.
```

Comment:
s/Craetes/Creates/

Also, "for or search" is a bit hard to read. Should be e.g., "for OR search"